docs/testing/webdriverio/01-overview.md

```diff
@@ -0,0 +1,35 @@
+---
+title: WebdriverIO Overview
+sidebar_label: Overview
+---
+
+# Overview
+
+WebdriverIO is a progressive automation framework built to automate modern web and mobile applications. It simplifies the interaction with your app and provides a set of plugins that help you create a scalable, robust and stable test suite.
+
+Testing with WebdriverIO has the following advantages:
+
+- __Cross Browser Support__: WebdriverIO is designed to support all platforms, either on desktop or mobile. You can run tests on actual browser your users are using, including covering different versions of them.
+- __Real User Interaction__: Interacting with elements in WebdriverIO through the WebDriver protocol is the closes you can get to reality compared to emulated interaction in virtual DOM environments.
```

Comment:
```suggestion
- __Real User Interaction__: Interaction with elements in WebdriverIO through the WebDriver protocol is much closer to native user-triggered interactions than what can be achieved with emulated DOM environments (such as JSDom or Stencil's own Mock-Doc).
```

Just a rewording suggestion - I'm thinking we probably want to keep the `virtual dom` terminology to refer to the rendering technique and not 'DOM implementations or mocks in JS', which is I think what you are referring to here